### ISSUE TITLE
Google Device node Sensor

### ISSUE THREAD

**Issue by FireWizard52** [mikejac/node-red-contrib-google-smarthome#215]
Hello all,

I continued to configure the Sensor node and I noticed a number of issues.
As an example I selected two sensors, Particulate Matter, PM10 and also PM25.
See the following test flow:

![Screenshot_Google_Device_PM1](https://user-images.githubusercontent.com/46283323/147133484-17a7de9c-e6c8-436f-be61-d727c2c15d9e.png)

However, if I wanted to configure the Google Device node, I noted the following:

![Screenshot_Google_Device_PM4](https://user-images.githubusercontent.com/46283323/147133889-ef619258-ccd5-4321-a6c9-f2fa6af1f5a7.png)

So, if you choose " Show trait" and select "Selected", you will see that there is nothing to select.

In the previous, I think, version it was possible:

![Screenshot_Google_Device_PM5](https://user-images.githubusercontent.com/46283323/147134160-30a4d08d-11f0-479f-97ee-ae1346e313f9.png)

A second thing I noted that also here 'If msg arrives on input, pass through to output:" does not give an output.
This has been reported before and I believe it has been picked up.

Regards

**Comment by ckhmer1** [comment]
I'll check this issue as soon as possible

**Comment by FireWizard52** [comment]
I have been able to experiment a little more and came across the following:

If I select as trait "Recommended" or "All".  I'm able to configure the sensor (and possible also other devices). If I return later and if I select "Selected" I see my configuration and I can modify this, if I wanted.
This makes sense and I believe that this is also the intended use. You cannot select "Selected" before you have it configured.
I think this is the intended use and if you agree, we can close this issue.

However I suggest, that the option "Selected" should be omitted, until one has configured the trait. Or at least a clear explanation in its README file.

Regards

**Comment by ckhmer1** [comment]
For every device type there are some traits mandatory, you cannot unselected then. Selecting select the first time, the mandatory traits should be shown.
I've to check if the device type sensor has mandatory traits, I guess yes, in this case I've to fix it.

**Comment by FireWizard52** [comment]
Agree,
I had seen that the sensor node had no mandatory traits.
So your explanation is acceptable to me.

But if a less experieced user want to use this node it is maybe confusing and therefore I suggested to remove that option, if there are no mandatory traits.

Thanks for publicing the new version 0.2.0.
Something to try tomorrow.

**Comment by FireWizard52** [comment]
I tested the sensor node and I noted that it works correctly and gives the Temperature in degrees Celsius as, e.g. TC: 9.8°C
Nice  :) 
However if I take other sensors such as PM10 or PM2.5, I don't see anything. I even cannot find the proper wording to ask for the values.
In the mean time I have transferred all the "old" and now deprecated nodes to the Google Device nodes and everything works as before.

I think we can close this issue.
